.travis.yml

```diff
@@ -26,23 +26,23 @@ cache:
     - $HOME/.composer/cache/files
 
 env:
-  matrix: SYMFONY_VERSION=4.0.*
+  matrix: SYMFONY_VERSION=4.2.*
   global:
-    - SYMFONY_DEPRECATIONS_HELPER="/.*each.*/"
+    - SYMFONY_DEPRECATIONS_HELPER="weak"
     - SYMFONY_PHPUNIT_DIR=.phpunit SYMFONY_PHPUNIT_REMOVE="symfony/yaml"
-    - SYMFONY_PHPUNIT_VERSION=6
-    - TEST_INSTALLATION=false
+    - KERNEL_CLASS=Symfony\Cmf\Bundle\RoutingBundle\Tests\Fixtures\App\Kernel
+    - SYMFONY_PHPUNIT_VERSION=5.7.26
 
 matrix:
   include:
-    - php: 7.2
-      env: SYMFONY_VERSION=4.0.*
-    - php: 7.1
-      env: COMPOSER_FLAGS="--prefer-lowest" SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=weak
-    - php: 7.2
-      env: SYMFONY_VERSION=3.3.*
-    - php: 7.2
-      env: SYMFONY_VERSION=3.4.*
+    - php: "7.3"
+      env: STABILITY="dev" SYMFONY_VERSION="4.2.*"
```

Comment:
why build everything with stability dev? one build with dev is fine for me, but the rest should be stable

Reply:
i thought, we build with dev when having the a point where we are on dev in master? So, if we require a dev version for our testing component i.e.?

Reply:
no, we want one build with dev to see if future symfony versions etc will pose some regression. but otherwise stable dependencies make more sense, they are what our users will have.

additionally, using dev can also hide problems when no stable release of a dependency is available.

i'd prefer to leave this as it was.